boards/arm/thingy91_nrf9160/CMakeLists.txt

```
@@ -7,15 +7,16 @@
 zephyr_library()
 zephyr_library_sources_ifdef(CONFIG_ADP536X adp5360_init.c)
 
-if(CONFIG_BOARD_THINGY91_NRF9160_NS)
-	# Use static partition layout to ensure the partition layout remains
-	# unchanged after DFU. This needs to be made globally available
-	# because it is used in other CMake files.
-	if(CONFIG_LWM2M_CARRIER)
-		set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_LIST_DIR}/thingy91_pm_static_lwm2m_carrier.yml CACHE INTERNAL "")
-	else()
-		set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_LIST_DIR}/thingy91_pm_static.yml CACHE INTERNAL "")
-	endif()
+if(CONFIG_THINGY91_STATIC_PARTITIONS_FACTORY)
+	set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_LIST_DIR}/thingy91_pm_static.yml CACHE INTERNAL "")
```

we don't use tabs in CMake files, please convert to two spaces.
```suggestion
  set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_LIST_DIR}/thingy91_pm_static.yml CACHE INTERNAL "")
```